src/conversions/std/osstr.rs

```diff
@@ -64,6 +64,17 @@ impl<'py> IntoPyObject<'py> for &OsStr {
     }
 }
 
+impl<'py> IntoPyObject<'py> for &&OsStr {
```

Comment:
All of these `&&` implementations are interesting. I think if we were starting again we wouldn't offer them, but due to the existing `ToPyObject` implementations they already exist. I wonder if some time in the future we should remove these (presumably would be a micro-optimization to avoid the pointer-to-slice), but doesn't seem worth doing with all the other churn in 0.23